gui/tsconfig.json

```diff
@@ -19,7 +19,33 @@
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+
+    /* Imports */
+    "baseUrl": "./src",
+    "paths": {
+      "@SpComponents/*": [
+        /* This implies a certain reorganization/recollection process */
+        "app/components/*",
+        "app/FileEditor/*",
+        "app/RunPanel/*",
+        "app/SamplerOutputView/*",
+        "app/SamplingOptsPanel/*",
+        "app/TabWidget/*"
+      ],
+      "@SpCore/*": ["app/Project/*", "app/gists/*"],
```

Comment:
Is it possible to keep sub-paths in this setup?

For example, I think I would ideally have everything under Project/ directly under SpCore, but keep the /gist/ in the path for those files, like SpCore/gist/...

Reply:
This is doable (and now done). Happy to look at other such things as well.